dipy/denoise/denspeed.pyx

```diff
@@ -307,7 +283,7 @@ cdef cnp.npy_intp copy_block_3d(double * dest,
 
     for i in range(I):
         for j in range(J):
-            memcpy(&dest[i * J * K + j * K], &source[i + min_i, j + min_j, min_k], K * sizeof(double))
+            memcpy(& dest[i * J * K  + j * K], & source[i + min_i, j + min_j, min_k], K * sizeof(double))
```

Comment:
No need for space after &. I know this is a bit puzzling. Because those operators are from C in a way you can consider them as part of the variable name.